File: Server/Server/src/Core/Application/Catalog/Customers/CustomerDto.cs
namespace Server.Application.Catalog.Customers;

public class CustomerDto : IDto
{
    public Guid Id { get; set; }
    public string Name { get; set; } = default!;
    public string? Code { get; set; }
    public string? TaxNumber { get; set; }
    public string? TaxOffice { get; set; }
    public bool Status { get; set; }
    public string? EntCode { get; set; }
}

File: client/src/app/models/customer/customerDto.ts
export class CustomerDto {
    id: string;  // Guid türü JavaScript'te string olarak temsil edilir
    name: string;
    code?: string;
    taxNumber?: string;
    taxOffice?: string;
    status:boolean;
    entCode?:string;

    constructor(id: string, name: string, code?: string, taxNumber?: string, taxOffice?: string, status?:boolean,entCode?:string) {
        this.id = id;
        this.name = name;
        this.code = code;
        this.taxNumber = taxNumber;
        this.taxOffice = taxOffice;
        this.status=status;
        this.entCode;
    }
}
